__tests__/__e2e__/cli.e2e.js

```diff
@@ -22,23 +22,28 @@ describe('End-to-End CLI', () => {
       })
     } catch (err) {
       expect(err.code).toBe(2) // means that vulnerabilities were found
+      console.log(err)
+      console.log(err && err.stdout)
+      console.log(err && err.stderr)
```

Comment:
Are you logging the err details for future tests debugging?

Reply:
No, just for this run to figure out why the windows PR times out